pkg/auth/model/model.go

```diff
@@ -217,23 +201,19 @@ func (s *Statements) Scan(src interface{}) error {
 	return json.Unmarshal(data, s)
 }
 
-func UserFromProto(pb *UserData) *User {
-	return &User{
-		ID: string(pb.Id),
-		BaseUser: BaseUser{
-			CreatedAt:         pb.CreatedAt.AsTime(),
-			Username:          pb.Username,
-			FriendlyName:      &pb.FriendlyName,
-			Email:             &pb.Email,
-			EncryptedPassword: pb.EncryptedPassword,
-			Source:            pb.Source,
-		},
+func UserFromProto(pb *UserData) *BaseUser {
+	return &BaseUser{
+		CreatedAt:         pb.CreatedAt.AsTime(),
+		Username:          pb.Username,
+		FriendlyName:      &pb.FriendlyName,
+		Email:             &pb.Email,
+		EncryptedPassword: pb.EncryptedPassword,
+		Source:            pb.Source,
```

Comment:
The property ExternalID is missing here compared to the BaseUser

Reply:
Nice catch! :)